.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is.)
+
+## To Reproduce
+
+(Write your steps here:)
+
+1. Step 1...
+1. Step 2...
+1. Step 3...
+
+## Expected behavior
+
+(Write what you thought would happen.)
+
+## Actual Behavior
+
+(Write what happened. Add screenshots, if applicable.)
+
```

Comment:
What about adding these as well : 

-  **Reproducibility**: [always, sometimes, random]
-  **Severity**: [feature, minor, major, crash, block]

Reply:
Yeah, we can add but I feel like it's overkilling of the template because this repository is small and after going through other organizations I did not find anything similar to what you have suggested. Also, there is no chance of a crash, block, etc because this is just a Wechaty gRPC service, not the whole product so we can avoid using these tags.